pkg/assembler/backends/arangodb/path_test.go

```diff
@@ -983,3 +983,1832 @@ func Test_Nodes(t *testing.T) {
 		})
 	}
 }
+
+func Test_Neighbors(t *testing.T) {
```

Comment:
all the unit tests for all the nouns have been consolidated here to make it easier to test without repeating multiple times. It still contains full coverage of all the noun tries.